internal/client/client.go

```diff
@@ -66,7 +66,10 @@ func (c *Client) DoRequest(endpoint string, target interface{}, queryParams ...m
 	values := c.URL.Query()
 	for _, m := range queryParams {
 		for k, v := range m {
-			values.Add(k, v)
+			// TODO: is there a better way to do this?
+			for _, j := range strings.Split(v, ",") {
+				values.Add(k, j)
+			}
```

Comment:
```suggestion
			for _, j := range strings.Split(v, ",") {
				values.Add(k, j)
			}
```
is this change okay? or will i need to replicate the `DoRequest` function and call it like `DoRequestMulti`

Reply:
Longer term we should really just adjust the function signature to accept `map[string][]string`. That's my fault, at the time none of the existing collectors had a need for multi-value keys. I'm happy for this to land as is if you like, I can fast-follow with the signature change.

Reply:
This shouldn't break anything as from quick testing its just like any other language and splitting without a match, just returns what you provided.

I guess the one downside is there will be an an extra single loop for all calls but i doubt that will make much of an impact

![image](https://github.com/onedr0p/exportarr/assets/5182053/53a10293-6b92-4a3e-91f3-6228f649c1be)

Reply:
though, there is the risk that if the params contain a , for whatever reason itll backfire.

If that does happen we can just change the code to join ids and split on something else instead if that ever happens